streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -50,26 +50,23 @@
 import static org.apache.kafka.streams.processor.internals.Task.State.RESTORING;
 
 public class TaskManager {
-    // initialize the task list
-    // activeTasks needs to be concurrent as it can be accessed
-    // by QueryableState
```

Comment:
Seems the comment does not make sense any longer? Only side cleanup in this class